### PR TITLE
More awesome

### PR DESCRIPTION
**Pull request by vweevers**
*Do not merge yet*. Opening this to collect feedback.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,76 +1,208 @@
...
-| **[level](https://github.com/level/level)** | [![dependencies](https://david-dm.org/Level/level.svg)](https://david-dm.org/level/level)| convenience package that bundles **[LevelUP](https://github.com/level/levelup)** and  **[LevelDOWN](https://github.com/level/leveldown)**
-| **[level-iterator-stream](https://github.com/level/iterator-stream)** | [![dependencies](https://david-dm.org/Level/iterator-stream.svg)](https://david-dm.org/level/iterator-stream) | Turn a AbstractDown iterator into a readable stream
-
-Other
-------------
+# Awesome Level ![level badge][level-badge]
+
+> A curated list of amazingly awesome **[Level]** modules and resources. [Add yours!](#contributing)
+
+## Core
```
Comment (vweevers) [inline README.md]: Does this header make sense?

File: README.md [hunk 2]
```diff
@@ -1,76 +1,208 @@
...
+
+| name | dependencies | description
+|:---- |:------------ |:-----------
+|**[`level`]** | [![david][120]][121] | Main entry point for beginners.<br> A convenient bundle of [`levelup`], [`leveldown`] and [`encoding-down`].
+|**[`levelup`]** | [![david][122]][123] | The glue that holds everything together.<br> Use this instead of [`level`] when you need a [custom store](#stores).
```
Comment (vweevers) [inline README.md]: Suggestions for a better description?

File: README.md [hunk 3]
```diff
@@ -1,76 +1,208 @@
...
 Level is licensed under the MIT license. All rights not explicitly granted in the MIT license are reserved. See the included `LICENSE.md` file for more details.
+
+<!-- Manual badges for now -->
+[down2.7]: https://img.shields.io/badge/abstract--leveldown-2.7-brightgreen.svg
+[down2.6]: https://img.shields.io/badge/abstract--leveldown-2.6-orange.svg
+[down2.4]: https://img.shields.io/badge/abstract--leveldown-2.4-orange.svg
+[down2.1]: https://img.shields.io/badge/abstract--leveldown-2.1-orange.svg
+[down1.0]: https://img.shields.io/badge/abstract--leveldown-1.0-red.svg
+[down0.x]: https://img.shields.io/badge/abstract--leveldown-0.x-red.svg
+[nondown]: https://img.shields.io/badge/abstract--leveldown-non-lightgrey.svg
```
Comment (vweevers) [inline README.md]: "non"  => "invalid"

File: README.md [hunk 4]
```diff
@@ -1,76 +1,208 @@
...
...cuYm9oZW1pYW5jb2RpbmcuY29tL3NrZXRjaCk8L2Rlc2NyaXB0aW9uPiAgICA8ZGVmcz48L2RlZnM%2BICAgIDxnIGlkPSJQYWdlLTEiIHN0cm9rZT0ibm9uZSIgc3Ryb2tlLXdpZHRoPSIxIiBmaWxsPSJub25lIiBmaWxsLXJ1bGU9ImV2ZW5vZGQiIHNrZXRjaDp0eXBlPSJNU1BhZ2UiPiAgICAgICAgPHBhdGggZD0iTTk0LDU0IEwwLDEwMC40MTAwNDIgTDAsMTk0LjgwMzM0NyBMOTQsMjQyIEwxODgsMTk0LjgwMzM0NyBMMTg4LDEwMC40MTAwNDIgTDk0LDU0IFoiIGlkPSJSZWN0YW5nbGUtMSIgZmlsbD0iI0Q0RUI5NSIgc2tldGNoOnR5cGU9Ik1TU2hhcGVHcm91cCI%2BPC9wYXRoPiAgICAgICAgPHBhdGggZD0iTTk0LDM5IEwwLDg1LjQxMDA0MTggTDAsMTc5LjgwMzM0NyBMOTQsMjI3IEwxODgsMTc5LjgwMzM0NyBMMTg4LDg1LjQxMDA0MTggTDk0LDM5IFoiIGlkPSJSZWN0YW5nbGUtMSIgZmlsbD0iIzk2REM3NSIgc2tldGNoOnR5cGU9Ik1TU2hhcGVHcm91cCI%2BPC9wYXRoPiAgICAgICAgPHBhdGggZD0iTTk0LDEgTDAsNDcuNDEwMDQxOCBMMCwxNDEuODAzMzQ3IEw5NCwxODkgTDE4OCwxNDEuODAzMzQ3IEwxODgsNDcuNDEwMDQxOCBMOTQsMSBaIiBpZD0iUmVjdGFuZ2xlLTEiIG9wYWNpdHk9IjAuNTUyODQwOTA5IiBmaWxsPSIjMzE3MzQyIiBza2V0Y2g6dHlwZT0iTVNTaGFwZUdyb3VwIj48L3BhdGg%2BICAgICAgICA8cGF0aCBkPSJNOTQsOTEuMTc3Nzc5MSBMMCw0NyBMMCwxNDEuNjY2NjY3IEw5NCwxODkgTDE4OCwxNDEuNjY2NjY3IEwxODgsNDcgTDk0LDkxLjE3Nzc3OTEgWiIgaWQ9IlJlY3RhbmdsZS0xIiBvcGFjaXR5PSIwLjU1Mjg0MDkwOSIgZmlsbD0iIzM0OTU0QyIgc2tldGNoOnR5cGU9Ik1TU2hhcGVHcm91cCI%2BPC9wYXRoPiAgICA8L2c%2BPC9zdmc%2B
```
Comment (vweevers) [inline README.md]: Put this badge SVG on leveldb.org?

File: README.md [hunk 5]
```diff
@@ -1,76 +1,208 @@
...
+[72]: https://img.shields.io/david/Level/deferred-leveldown.svg
+[73]: https://david-dm.org/level/deferred-leveldown
+[74]: https://img.shields.io/david/mvayngrib/cachedown.svg
+[75]: https://david-dm.org/mvayngrib/cachedown
+
+## Plugins for [`levelup`]
+
+Modules that add functionality to [`levelup`].
+
+| name | dependencies | description
```
Comment (vweevers) [inline README.md]: Update these table headers (same below)

File: README.md (hunk 1 above)
Comment (MeirionHughes) [reply README.md]: maybe drop the the "amazingly". Other than that its fine.

Comment (vweevers) [reply README.md]: Sorry I was referring to "Core". I like "amazingly awesome" :)

File: README.md (hunk 2 above)
Comment (MeirionHughes) [reply README.md]: Seems fine. 👍

File: README.md (hunk 4 above)
Comment (MeirionHughes) [reply README.md]: Perhaps needs to be the awesome badge here: see https://github.com/sindresorhus/awesome/blob/master/create-list.md

Comment (vweevers) [reply README.md]: IMO the awesome badge is rather ugly. Too much "bro" and doesn't match the standard badge style

**Comment by ralphtheninja** [comment]
This is turning into awesome badassery :)

**Pull request merged**